src/platforms/hosted/README.md

```diff
@@ -1,116 +1,133 @@
-# PC-Hosted BMP
-Compile in src with "make PROBE_HOST=hosted". This needs minimal external
-support.  "make PROBE_HOST=hosted HOSTED_BMP_ONLY=0" will compile support for FTDI,
-STLink, CMSIS-DAP and J-Link probes, but requires external libraries.
+# Black Magic Debug App (BMDA)
+
+Compile with:
+
+```sh
+meson setup build
+meson compile -C build
+```
```

Comment:
It'd be more readable to remove this and integrate it with Description as shown below.